lib/OptimizationOptimisers/src/OptimizationOptimisers.jl

```diff
@@ -59,9 +59,19 @@ function SciMLBase.__solve(cache::OptimizationCache{
     else
         cache.solver_args.epochs
     end
+    maxiters = if cache.solver_args.maxiters === nothing
+        if cache.solver_args.epochs === nothing
+            throw(ArgumentError("The number of iterations must be specified with either the epochs or maxiters kwarg. Where maxiters = epochs*length(data)."))
+        else
+            cache.solver_args.epochs * length(data)
+        end
+    else
+        cache.solver_args.maxiters
+    end
 
     epochs = Optimization._check_and_convert_maxiters(epochs)
-    if epochs === nothing
+    maxiters = Optimization._check_and_convert_maxiters(maxiters)
+    if epochs === nothing || maxiters === nothing
```

Comment:
```suggestion
    if epochs === nothing & maxiters === nothing
```

This should be an `and` here, right?

Reply:
After checking and finding the right numbers, neither of them should be null.

Reply:
Ah yeah, I missed the inner else branch there, makes sense.